lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -119,7 +127,11 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :next_version,
                                        description: "Next version of the SDK",
                                        optional: true,
-                                       type: String)
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :automatic_release,
+                                       description: "If this is an automatic release",
+                                       optional: true,
+                                       type: Boolean)
```

Comment:
Actually, according to fastlane's documentation, boolean parameters should be set with `is_string: false` instead of specifying the type: https://docs.fastlane.tools/advanced/actions/